### DISCUSSION TITLE
Error updating Jellyfin LXC to newest 10.9.11 from 10.9.9

### DISCUSSION THREAD

**Discussion by drwkabbov** [tteck/Proxmox#3795] [Help Needed] [answered]
Hi I have a Jellyfin LCX container created using the helper script. Today I tried to watch something and was having errors so I checked if there was an updated version of Jellyfin server. There was an update from my version 10.9.9 to a new 10.9.11, i tried to run update within Jellyfin LXC and came away with the following error  

- Updating Jellyfin LXC   
[ERROR] in line 60: exit code 0: while executing command apt-get -y upgrade &> /dev/null

the LXC has not updated. Has this been seen before? anything other then deleting LXC and starting again? I want to avoid that as I have setup unprivileged pass through and mounting NFS shares to the container.

**Comment by xekil** [comment, marked as answer]
Hello, try

`apt update && apt upgrade`

**Comment by drwkabbov** [comment]
Hi, thanks for the tip. I finally got to testing the update via "apt update && apt upgrade" however after getting to the do you wish to proceed, selecting yes gets me the following error.

Preconfiguring packages ...
dpkg: unrecoverable fatal error, aborting:
 unknown system group 'ssl-cert' in statoverride file; the system group got removed
before the override, which is most probably a packaging bug, to recover you
can remove the override manually with dpkg-statoverride
E: Sub-process /usr/bin/dpkg returned an error code (2) 

It says i can manually override however i would have no idea what i would be doing.
i have deleted nothing after the script install, all i have allowed is unprivileged HW decode by following the following link 
https://forum.proxmox.com/threads/guide-jellyfin-remote-network-shares-hw-transcoding-with-intels-qsv-unprivileged-lxc.142639/ 

Is there away around this?
everything works at the moment however i am getting the warning that the tv app will not support the older version of jellyfin after the next update.
